app/views/chugs/_podium_list.html.erb

```diff
@@ -10,7 +10,8 @@
             <% end %>
             <div class="flex items-center justify-center">
               <p class="text-center absolute gold-text w-24 text-sm"><strong style="font-size: large"><%= chug.time.round(2) %></strong>s
-                <br><i class="text-sm"><%= chug.user.name %></i></p>
+                <br><i><%= chug.user.id == 7 && chug.comment.present? ? chug.comment : chug.user.name %></i>
```

Comment:
Je kan nu ook chug.extern? Gebruiken ipv chug.user == 7

Reply:
chug.extern is toch geen property in dit geval, maar een collection van chugs die van extern zijn en bij dat chugtype horen?